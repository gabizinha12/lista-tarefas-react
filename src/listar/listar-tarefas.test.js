import React from "react";
import ReactDOM from "react-dom";
import ListarTarefas from "../listar/listar-tarefas";
import MemoryRouter from "react-router";
import { shallow } from "enzyme";

import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("teste do componente de listagem de tarefas", () => {
  it("deve renderizar o componente sem erros", () => {
    const component = shallow(<ListarTarefas />);
    expect(component).toMatchSnapshot();
  });
});
