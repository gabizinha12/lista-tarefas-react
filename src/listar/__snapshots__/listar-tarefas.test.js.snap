// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teste do componente de listagem de tarefas deve renderizar o componente sem erros 1`] = `ShallowWrapper {}`;
